src/PIL/features.py

```diff
@@ -94,7 +95,7 @@ def get_supported():
     return ret
 
 
-def pilinfo(out=None):
+def pilinfo(out=None, brief=False):
```

Comment:
Suggestion: Instead of 'brief=False', what about 'include_python_version=True'?

Reply:
Or flip it and call it `verbose`?

Reply:
@radarhere The significant difference is whether the list of supported image formats is printed.

Thinking about it, I don't think it would be a bad idea to include the Python version in `selftest.py`. The parameter could then be called `supported_formats=True`.

Reply:
> Thinking about it, I don't think it would be a bad idea to include the Python version in `selftest.py`. The parameter could then be called `supported_formats=True`.

Done, see comment below.